graphics/InsightToolkit-devel/Portfile

```diff
@@ -1,77 +1,161 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           active_variants 1.1
+PortGroup           conflicts_build 1.0
 
 name                InsightToolkit-devel
-version             4.8.2
-revision            15
-categories          graphics devel
-platforms           darwin
+github.setup        InsightSoftwareConsortium ITK 5.3.0 v
+revision            0
+categories          graphics science devel
 license             Apache
+conflicts_build     InsightToolkit4
 
 maintainers         {stromnov @stromnov} openmaintainer
 
 description         Insight Segmentation and Registration Toolkit (ITK)
 
-long_description    Insight Segmentation and Registration Toolkit (ITK) \
-                    is an open-source, cross-platform system that provides \
-                    developers with an extensive suite of software tools \
-                    for image analysis.
+long_description    {*}${description} is an open-source, cross-platform system that \
+                    provides developers with an extensive suite of software tools \
+                    for image analysis. Developed through extreme programming \
+                    methodologies, ITK builds on a proven, spatially-oriented \
+                    architecture for processing, segmentation, and registration \
+                    of scientific images in two, three, or more dimensions.
 
-homepage            http://www.itk.org
-master_sites        sourceforge:itk
+homepage            https://itk.org
 
 distname            InsightToolkit-${version}
```

Comment:
Same comment here, vis-a-vis InsightToolkit4. (In terms of using `dist_subdir InsightToolkit`, to avoid duplication of distribution files.)